backend/lambda_functions/interview_count_lambda.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import requests
+from datetime import datetime, timedelta
+
+cloudwatch = boto3.client('cloudwatch')
+
+def lambda_handler(event, context):
```

Comment:
Please devide the code under the `lambda_handler` function into smaller functions for better readability.